airflow/www/views.py

```diff
@@ -3908,9 +3909,11 @@ class DagFilter(BaseFilter):
     """Filter using DagIDs."""
 
     def apply(self, query, func):
-        if get_airflow_app().appbuilder.sm.has_all_dags_access(g.user):
+        if get_auth_manager().is_authorized_dag(
+            method="GET", user=g.user
+        ) or get_auth_manager().is_authorized_dag(method="PUT", user=g.user):
             return query
```

Comment:
```suggestion
        if get_auth_manager().is_authorized_dag(method="GET", user=g.user):
            return query
        if get_auth_manager().is_authorized_dag(method="PUT", user=g.user):
            return query
```

I’m not a fan very long if statements, especially with how Black tends to wrap it very awkwardly.

Reply:
Sounds good to me 👍